ui/component/notification/index.js

```diff
@@ -1,8 +1,14 @@
 import { connect } from 'react-redux';
```

Comment:
When changing the active channel via "Set replying as", we'll need to re-fetch the reactions to get the right numbers (mine vs others).  Currently, it resets to zero.

Reply:
I ran into this but wasn't sure what happened - but now this makes sense.